src/detect-engine-dnp3.c

```diff
@@ -0,0 +1,78 @@
+/* Copyright (C) 2015 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+#include "suricata-common.h"
+#include "stream.h"
+#include "detect.h"
+#include "detect-engine-content-inspection.h"
+#include "detect-dnp3.h"
+#include "detect-engine-dnp3.h"
+#include "app-layer-dnp3.h"
+
+int DetectEngineInspectDNP3Data(ThreadVars *tv, DetectEngineCtx *de_ctx,
```

Comment:
For now code I would like these engines to go into the keyword files. No need to have them separately anymore (if there ever was).